edm4hep.yaml

```
@@ -416,68 +416,68 @@ datatypes :
     Members:
       - float weight                        // weight of this association
     OneToOneRelations :
-     - edm4hep::ReconstructedParticle  rec  // reference to the reconstructed particle
-     - edm4hep::MCParticle sim              // reference to the Monte-Carlo particle
+     - edm4hep::ReconstructedParticle from  // reference to the reconstructed particle
+     - edm4hep::MCParticle to               // reference to the Monte-Carlo particle
 
   edm4hep::MCRecoCaloAssociation:
     Description: "Association between a CaloHit and the corresponding simulated CaloHit"
     Author: "C. Bernet, B. Hegner"
     Members:
       - float weight                    // weight of this association
     OneToOneRelations:
-     - edm4hep::CalorimeterHit rec     // reference to the reconstructed hit
-     - edm4hep::SimCalorimeterHit sim  // reference to the simulated hit
+     - edm4hep::CalorimeterHit from     // reference to the reconstructed hit
+     - edm4hep::SimCalorimeterHit to    // reference to the simulated hit
```

```suggestion
     - edm4hep::CalorimeterHit to     // reference to the reconstructed hit
     - edm4hep::SimCalorimeterHit from    // reference to the simulated hit
```
The name of the datatype suggests that the MC part is first, so I think it is more intuitive for it to be the "from" part of the association, right? A similar argument can be made for some of the other associations as well.